features/handled_errors.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Handled Error
+
+Scenario: Handled Error report override
```

Comment:
I needed to look at the source code of the test to know what this test does. A better title would be more explicit, e.g. "Override errorClass and message from a notifyError() callback"